scripts/deploy_service

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+#
+# deploy containers to our ecs services tagged with the git digest of the current build
+
+set -u
+
+service_id="$1"
+image="${2:-}"
+
+function_name="$APP_ENV-ecs-manager"
+function_qualifier="master"
+ecr_backend="${AWS_ACCOUNT_ID}.dkr.ecr.us-west-2.amazonaws.com"
+
+if [[ -n "$image" ]]; then
+  image_tag="$ecr_backend/$image:${CIRCLE_SHA1-$(git rev-parse HEAD)}"
+fi
+
+_payload() {
+  local image_tag="${1:-}"
+  local cluster_id="${3:-$APP_ENV-easi-app}"
```

Comment:
For cluster_id, should the "3" be "2"?

Reply:
It should just be hard-coded really. This value doesn't change except with the `$APP_ENV`. I'll get it out next time I'm in here changing things, but for now it shouldn't affect anything functionally since we call this function exactly once and with one argument.

Reply:
Right, I figured it wasn't affecting anything functionally at the moment. I created EASI-474 to clean it up. Probably best to do that tomorrow while it's still fresh in our minds.